bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/bridge/BoschSHCBridgeHandler.java

```diff
@@ -0,0 +1,363 @@
+            ContentResponse contentResponse = httpClient.createRequest(url, GET).send();
+
+            String content = contentResponse.getContentAsString();
+            logger.debug("Response complete: {} - return code: {}", content, contentResponse.getStatus());
+
+            Type collectionType = new TypeToken<ArrayList<Device>>() {
+            }.getType();
+            ArrayList<Device> devices = gson.fromJson(content, collectionType);
+
+            if (devices != null) {
+                for (Device d : devices) {
+                    // Write found devices into openhab.log until we have implemented auto discovery
+                    logger.info("Found device: name={} id={}", d.name, d.id);
+                    if (d.deviceSerivceIDs != null) {
+                        for (String s : d.deviceSerivceIDs) {
+                            logger.info(".... service: {}", s);
+                        }
+                    }
+                }
+            }
+        } catch (InterruptedException | TimeoutException | ExecutionException e) {
+            logger.debug("HTTP request failed with exception {}", e.getMessage());
+            return false;
+        }
+
+        return true;
+    }
+
+    private void handleLongPollResult(LongPollResult result) {
+        for (DeviceStatusUpdate update : result.result) {
+            if (update != null && update.state != null) {
+                logger.debug("Got update for {}", update.deviceId);
+
+                boolean handled = false;
+
+                Bridge bridge = this.getThing();
+                for (Thing childThing : bridge.getThings()) {
+                    // All children of this should implement BoschSHCHandler
+                    ThingHandler baseHandler = childThing.getHandler();
+                    if (baseHandler != null && baseHandler instanceof BoschSHCHandler) {
+                        BoschSHCHandler handler = (BoschSHCHandler) baseHandler;
+                        String deviceId = handler.getBoschID();
+
+                        handled = true;
+                        logger.debug("Registered device: {} - looking for {}", deviceId, update.deviceId);
+
+                        if (deviceId != null && update.deviceId.equals(deviceId)) {
+                            logger.debug("Found child: {} - calling processUpdate with {}", handler, update.state);
+                            handler.processUpdate(update.id, update.state);
+                        }
+                    } else {
+                        logger.warn("longPoll: child handler for {} does not implement Bosch SHC handler", baseHandler);
+                    }
+                }
+
+                if (!handled) {
+                    logger.debug("Could not find a thing for device ID: {}", update.deviceId);
+                }
+            }
+        }
+    }
+
+    private void handleLongPollFailure(Throwable e) {
+        logger.error("Long polling failed", e);
```

Comment:
The error logging level should be reserved for issues catastrophic enough to threaten the operation of openHab itself. An error in a binding shouldn't qualify for this. Please read the [openHab coding guidelines](https://www.openhab.org/docs/developer/guidelines.html#f-logging) for expected use of logging in bindings.

Please change this to warn.

Reply:
Changed it to logger.warn